template-for-repository/proofs/Makefile.common

```diff
@@ -389,13 +389,13 @@ report2: $(LOGDIR)/result.xml $(LOGDIR)/property.xml $(LOGDIR)/coverage.xml
 ################################################################
 # Targets to clean up after ourselves
 clean:
-	$(RM) $(DEPENDENT_GOTOS)
-	$(RM) TAGS*
-	$(RM) *~ \#*
+	-$(RM) $(DEPENDENT_GOTOS)
+	-$(RM) TAGS*
+	-$(RM) *~ \#*
```

Comment:
Why the -?  The definition of RM = rm -f, so there is no failure to ignore when the file does not exist.